app/Controllers/entryController.php

```diff
@@ -190,7 +190,8 @@ public function readAction(): void {
 					'a' => 'index',
 					'params' => $params,
 				],
-				'readAction'
+				'readAction ',
+				(bool) FreshRSS_Context::userConf()->good_notification_timeout,
```

Comment:
@Alkarex @Frenzie @Inverle : could you please give me feedback?
Is it a good approach to add a bool parameter here? Is it okay how it is done?
I did this commit as example for verification if it is good or not. (back end development is not my best side :D)

Reply:
thank you very much :)